nipype/interfaces/spm/preprocess.py

```diff
@@ -845,6 +845,8 @@ class NewSegmentInputSpec(SPMCommandInputSpec):
                                         desc='mni, eastern, subj, none ')
     warping_regularization = traits.Float(field='warp.reg',
                                           desc='Aproximate distance between sampling points.')
+    warping_regularization_12 = traits.List(traits.Float(), minlen=5, maxlen=5, field='warp.reg',
```

Comment:
this should likely be either or with warping_regularization. a couple of alternatives:

you can make warping regularization use `traits.Either(traits.Float, traits.List(traits.Float, minlen, maxlen), desc= ...)`

that may keep the interface cleaner. in the description you can add that the list of 5 elements is only supported in SPM 12.